package.json

```diff
@@ -22,12 +30,8 @@
   "license": "MIT",
   "licenseFilename": "LICENSE",
   "readmeFilename": "README.md",
-  "peerDependencies": {
-    "react": "^16.8.1",
-    "react-native": ">=0.59.0-rc.0 <1.0.x"
-  },
   "devDependencies": {
-    "react": "^16.8.3",
-    "react-native": "^0.59.10"
+    "@types/react-native": "^0.66.11",
```

Comment:
Is there a reason to not go for a newer version of RN types?

Reply:
Good point, thanks!